src/test/java/seedu/guest/model/GuestPrefsTest.java

```diff
@@ -4,18 +4,17 @@
 
 import org.junit.jupiter.api.Test;
 
-public class UserPrefsTest {
+public class GuestPrefsTest {
 
     @Test
     public void setGuiSettings_nullGuiSettings_throwsNullPointerException() {
-        UserPrefs userPref = new UserPrefs();
+        GuestPrefs userPref = new GuestPrefs();
         assertThrows(NullPointerException.class, () -> userPref.setGuiSettings(null));
     }
 
     @Test
     public void setAddressBookFilePath_nullPath_throwsNullPointerException() {
```

Comment:
Should the method name be setGuestBookFilePath_nullPath_throwsNullPointerException?